packages/login/test/login-overlay.common.js

```diff
@@ -82,17 +81,6 @@ describe('opened overlay', () => {
     expect(overlay.opened).to.be.true;
   });
 
-  it('should fire `login` event', () => {
-    const loginSpy = sinon.spy(overlay, '_retargetEvent');
```

Comment:
This test fails because in case of LitElement, `_retargetEvent` function is passed to the template, but under the hood it's used to create an actual event listener (with `this` bound correctly).

Besides, there is already a proper test for the `login` event below, so we can remove this one.